MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/geometry/LatLngBounds.java

```diff
@@ -372,8 +372,9 @@ private boolean containsLatitude(final double latitude) {
   }
 
   private boolean containsLongitude(final double longitude) {
-    return (longitude <= this.longitudeEast)
-      && (longitude >= this.longitudeWest);
+    return ((longitude <= this.longitudeEast) && (longitude >= this.longitudeWest))
+            || ((longitude <= this.longitudeEast - 360.0) && (longitude >= this.longitudeWest - 360.0))
+            || ((longitude <= this.longitudeEast + 360.0) && (longitude >= this.longitudeWest + 360.0));
   }
```

Comment:
@tobrun I'm not totally sure about this modification, do you see any issues with it? since previously this method doesn't consider the case of the unwrapped bounds, will this cause unexpected behaviour?

Reply:
The strategy for this class was that it doesn't handle bounds wrapping, so this change might actually be unexpected.

Reply:
@LukasPaczos  thanks for the comment, so I guess I should leave it as it is now, and update my unit tests to unwrap the LatLngCoordinates before using the the contains method. 
Or is it better that I introduce a new method `containsLongitudeUnwrapped`  with the bounds wrapping support?

Reply:
I missed this change, this could indeed result in unexpected changes for users.

> update my unit tests to unwrap the LatLngCoordinates before using the the contains method.

That would be my preference for now, if there is an actual need for `containsLongitudeUnwrapped` we can revisit later.

Reply:
👍 Reverted changes in LatLngBounds#contains.